TiFBackendUtils/MySQLDriver/test/tableDefinitions.ts

```diff
@@ -20,18 +20,18 @@ export const tableDefintionsByFamily = [
     CREATE TABLE IF NOT EXISTS event (
     id bigint NOT NULL AUTO_INCREMENT,
     description varchar(1000) NOT NULL,
-    startDateTime datetime NOT NULL,
-    endDateTime datetime NOT NULL,
+    startDateTime datetime(3) NOT NULL,
```

Comment:
Remove extra precision in production environment. Check if mysql NEEDS precision to be defined in the table. In sqllite it's not necessary.